app/models/mixins/authentication_mixin.rb

```diff
@@ -288,7 +288,6 @@ def authentication_check_no_validation(type, options)
           [:error, err]
         end
       end
-
```

Comment:
Right.. if you do the `EmsMiddleware#index` action and it doesn't call `render` somehow, rails tries to load a template with this name.. so you can either keep it, or ensure that `index` always calls `render`, after which, the file shuldn't be needed..